src/test/java/org/mafagafogigante/dungeon/io/ResourcesFolderTest.java

```diff
@@ -0,0 +1,72 @@
+package org.mafagafogigante.dungeon.io;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+
+public class ResourcesFolderTest {
+
+    private static final String RESOURCES_TARGET_PATH = "../classes/";
+
+    private static final String JSON_FILE_EXTENSION = ".json";
+    private static final String FONT_FILE_EXTENSION = ".ttf";
+    private static final String SPLASH_SCREEN_FILE_NAME = "splash.png";
+
+    private File resourcesDir;
+
+    @Before
+    public void initialize() {
+
```

Comment:
I don't understand this and the other few blank lines at the beginning of the methods. I would remove them.